Update Navigation Bar

lawtlee opened the pull request:
Updated the Navbar to follow the order
given by PageOrder in globalTypes.ts

Update Navbar Navigation Order

## Summary

Closes #305

- Switched two divs in order to accurately represent the order of pages given by PageOrder
<!-- list any new dependencies required for this change -->

## Screenshots

<!--
    Add Screenshots of the feature in play, terminal pastes, etc. as necessary
-->

<img width="362" alt="image" src="https://github.com/uclaacm/parcel-pointers/assets/104863284/91871e66-cd57-40e6-b0e7-2f6ae973d723">

lawtlee committed:
Update Navigation Bar

Updated the Navbar to follow the order
given by PageOrder in globalTypes.ts

Dankco commented:
Could you decrease the spacing here a little bit between the "Why Pointers?" and "Demo" and "Demo" and "Lessons"? Maybe more proportional to the space between Lessons and Exercises.
![image](https://github.com/uclaacm/parcel-pointers/assets/78634083/c1462834-3f5a-48e3-bd94-236551443698)

lawtlee committed:
Update Sidebar.scss

Changed the dropdown-header to flex instead of block
allowed for all headings to be centered vertically

lawtlee committed:
Update Sidebar.scss

Fixed the lint error, i didn't alphabetically
order the css

Dankco reviewed (approved):
Looks good!
